scripts/Calibration/tube_calib.py

```diff
@@ -223,8 +223,7 @@ def getPoints(integrated_ws, func_forms, fit_params, which_tube, show_plot=False
             peak_index = fit_edges(fit_params, i, get_points_ws, calib_points_ws)
         else:
             peak_index = fit_gaussian(fit_params, i, get_points_ws, calib_points_ws)
-
-        peak_centre = mtd[calib_points_ws + '_Parameters'].row(peak_index).items()[1][1]
+        peak_centre = tuple(mtd[calib_points_ws + '_Parameters'].row(peak_index).items())[1][1]
```

Comment:
I think it's recommended that we use `ADS.retrieve()` instead of `mtd[]` in the main code. Could this be changed while we're here?